docs/user/dashboard/lens-advanced.asciidoc

```diff
@@ -39,6 +39,79 @@ image::images/lens_end_to_end_1_1.png[New visualization popover]
 
 . Make sure the *kibana_sample_data_ecommerce_* index appears.
 
+[discrete]
+[[add-a-data-layer]]
+==== Add a second chart type or index pattern to your existing chart
```

Comment:
Did you add this to the very top on purpose? It seems pretty out of place here (e.g. `To add a line chart layer on top of your area chart` - the user didn't configure an area chart yet, right?)

Also, do we need to adjust the screenshots for this?

Reply:
I moved this out of the intro tutorial into the more advanced tutorial because it's my opinion that multi-layer visualizations are less common than single-layer charts. Happy to talk about this in offline since I'm not sure this is right!